.github/workflows/test.yaml

```diff
@@ -0,0 +1,39 @@
+name: Run tests
+
+on:
+  push:
+    branches:
+      - oheed
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.9'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r requirements.txt
+          pip install pytest
+          pip install pylint
+          pip install black
+      - name: Run tests
+        run: pytest tests/testcases.py
+      - name: Run pylint
+        run: pylint std/stdio.py
+      - name: run Black
+        run: black std/stdio.py
```

Comment:
Should run them for the whole codebase

Reply:
ok, I am working on adding this for whole project.